mobile/library/java/io/envoyproxy/envoymobile/engine/EnvoyConfiguration.java

```diff
@@ -169,30 +171,20 @@ public EnvoyConfiguration(
     this.enablePlatformCertificatesValidation = enablePlatformCertificatesValidation;
     this.enableSkipDNSLookupForProxiedRequests = enableSkipDNSLookupForProxiedRequests;
   }
-
   /**
-   * Resolves the provided configuration template using properties on this
-   * configuration.
+   * Creates configuration YAML based on the configuration of the class
    *
-   * @param configTemplate the template configuration to resolve.
-   * @param platformFilterTemplate helper template to build platform http filters.
-   * @param nativeFilterTemplate helper template to build native http filters.
-   * @param altProtocolCacheFilterInsert helper insert to include the alt protocol cache filter.
-   * @param gzipFilterInsert helper to include to enable gzip compression.
-   * @param brotliFilterInsert helper to include to enable brotli compression.
-   * @param socketTagFilterInsert helper to include to enable socket tagging.
-   * @param persistentDNSCacheConfigInsert helper to include to enable DNS cache.
-   * @param certValidationTemplate helper template to enable cert validation.
```

Comment:
Yay! These arguments always felt a bit unnecessary as their values live in config.cc.